godot-macros/src/lib.rs

```diff
@@ -300,6 +300,46 @@ use crate::util::{bail, ident, KvParser};
 /// }
 /// ```
 ///
+/// To declare groups and subgroups append `group` key to a `#[export]` attribute. Fields without group will be exported first,
+/// followed by properties with group only, tailed by ones with group & subgroup declarations. Relative order of fields is being preserved
```

Comment:
> Relative order of fields is being preserved

Maybe mention that this applies within the same group/subgroup, as well as on top level.